collaboration-and-sharing/public-shareable-links.md

```diff
@@ -0,0 +1,31 @@
+# Create public shareable links
+
+{% hint style="info" %}
+**Enterprise plan only**
+This feature is only available in custom enterprise pricing plans.
+{% endhint %}
+
+When an asset or media collection needs to be shared externally with someone outside your ImageKit environment, a public shareable link can be created for it. Public links allow limited, read-only access to your assets without the need to register for an ImageKit account. This includes the ability to download multiple assets in a ZIP file. A link can be created for a file, folder, or media collection. Configurable options for a public link include password protection, a validity timeframe, and whether to expose [draft assets](../media-library/overview/draft-assets.md) or not. If an expiration date is set for a public link, it will be automatically deleted after expiration.
```

Comment:
Docs should be following a structure-

- Explain what are public links and why they are helpful/use cases.
- What all actions can be done on public links? Create, edit, view, delete sections.
- Subsection on creating public links and all the options available. (Validity, exposing draft assets)
- Similarly for view (password, private files, signed URL setting behaviour etc), edit, delete (Full access users can list all in settings). Then a section for restricted users.
-  Audit logs usage - helpful in compliance or tracking.
- What a public user can do (fetch, see file metadata, download)

Reply:
reorganized